### DISCUSSION TITLE
Aliasing a color returns default shade list

### DISCUSSION THREAD

**Discussion by driss-chelouati** [tailwindlabs/tailwindcss#9774] [Help]
Let's say I want to alias a color that I have extended:

```
const colors = require('tailwindcss/colors')

module.exports = {
     theme: {
         extend: {
              colors: {
                  slate: {
                     1000: '#0a101f',
                  },
                  muted: colors.slate,
              }
         }
     }
}
```

In my templates, I can use **bg-muted-*** (where * ranges from 50 to 900) but I cannot use **bg-muted-1000**. However, **bg-slate-1000** works. This use to work before upgrading to 3.2.2. Did I miss something?

**Comment by wongjn** [comment]
The change in behavior could be related to #9294 but I'm not sure – where is the `colors` variable coming from?

**Reply by driss-chelouati** [reply]
The `colors` variable comes from a require statement at the top of the file, getting it from the tailwindcss package:

```
const colors = require('tailwindcss/colors')
```
